Removed Linux build

uilianries opened the pull request:
- Removed MingW build
- Removed Linux job on Circle CI

uilianries committed:
Add XCode 7 on Circle Ci

- Added xcode 7 on Circle CI 2.0
- Removed MingW build
- Removed Linux job on Circle CI

Signed-off-by: Uilian Ries <uilianries@gmail.com>

solvingj commented:
might as well take the macos builds out of travis.yml. 

also, we should add (windows), (linux), (mac) in the grid with the badges.  i'll play with that and get the formatting right.

uilianries committed:
Separate Linux and OSX build by CI tool

- Now Travis only builds Linux job, and Circle CI will
  run OSX.
- Updated CI badges

Signed-off-by: Uilian Ries <uilianries@gmail.com>

uilianries committed:
Fix Appveyor build

- I forgot to install bincrafters tools

Signed-off-by: Uilian Ries <uilianries@gmail.com>
